x-pack/plugins/viz_editor/public/pseudo_plugins/xy_chart_plugin/xy_chart_plugin.tsx

```diff
@@ -172,7 +181,7 @@ function getSuggestionsForField(
         q1: {
           datasourceRef: datasourceName,
           select: [
-            { ...firstOperation, alias: field.name },
+            { operation: 'column', alias: field.name, argument: { field: field.name } },
```

Comment:
This logic breaks suggestions- instead of making this change, please instead modify `field_config.ts` to put `column` first.

Reply:
Good point.